wrapper/src/test/java/integration/host/AuroraMysqlContainerTest.java

```diff
@@ -163,7 +163,7 @@ static void setUp() throws SQLException, InterruptedException, UnknownHostExcept
       container.start();
     }
     mysqlProxyPort =
-        containerHelper.createInstanceProxies(
+        containerHelper.getProxyPort(
```

Comment:
I'm not sure the name gets better. getProxyPort() gives an idea what it returns but it doesn't highlight that a container instance is created behind the scene. 

createInstanceProxiesAndReturnPort() ?

Reply:
Ah, I didn't realize this method also creates proxies, I thought it was just determining the proxy port. I'll change the name